www/src/components/headSeo.astro

```diff
@@ -20,27 +20,17 @@ const imageAlt = frontmatter.image?.alt ?? OPEN_GRAPH.image.alt;
 <meta property="og:title" content={formattedContentTitle} />
 <meta property="og:type" content="article" />
 <meta property="og:url" content={canonicalUrl} />
-<meta
-  property="og:locale"
-  content={frontmatter.ogLocale ?? SITE.defaultLanguage}
```

Comment:
these checks came from the old astro config which didn't have any typesafety. we dont need it anymore